### PR TITLE
Add URLs to HF MLX-Community org.

### PR DESCRIPTION
**Pull request by Vaibhavs10**

### CODE REVIEW

File: llama/README.md [hunk 1]
```diff
@@ -20,10 +20,8 @@ weights you will need to request access from Meta:
 - [Request Llama v1](https://docs.google.com/forms/d/e/1FAIpQLSfqNECQnMkycAp2jP4Z9TFX0cGR4uf7b_fBxjY_OjhJILlKGA/viewform)
 - [Request Llama v2](https://ai.meta.com/resources/models-and-libraries/llama-downloads/)
 
-
-Alternatively, you can also download a select converted checkpoints from the
-[mlx-llama](https://huggingface.co/mlx-llama) community organisation on Hugging
-Face and skip the conversion step.
+> [!TIP]
+> Alternatively, you can also download a few converted checkpoints from the [mlx-community](https://huggingface.co/mlx-community) community organisation on Hugging Face and skip the conversion step.
```
Comment (awni) [inline llama/README.md:24]: Should we go `MLX Community` for all of the refs to be consistent with the main README?

Comment (Vaibhavs10) [reply llama/README.md:24]: Yess! let me update real quick

**Comment by Vaibhavs10** [comment]
@awni - Should be all standardised now! 🤗

**Review by awni — APPROVED** [review]
Awesome 🤗

**Pull request merged**